yeelight/lib/yee.js

```diff
@@ -202,6 +203,21 @@ YeeDevice = function (did, loc, model, power, bri,
             return;
         }
 
+        if (this.model == "ceiling"){
+            if(val<=10){
+                console.log('start moonlight scene');
+                var req = {id:1, method:'set_scene',
+		            params:['nightlight', 1]};
+	            this.sendCmd(req);
+                return;
+            }else{
+                console.log('start daylight scene');
+                var req = {id:1, method: 'set_ct_abx', params: [5500, 'smooth', 500]}
```

Comment:
I don't think change color temperature is a good idea here. Why not just change brightness?